metadata-ws/src/main/resources/application.properties

```diff
@@ -46,3 +46,7 @@ swagger.apiinfo.license=Apache 2.0
 swagger.apiinfo.licenseUrl=http://www.apache.org/licenses/LICENSE-2.0
 swagger.apiinfo.version=@project.version@
 swagger.apiinfo.termsOfServiceUrl=
+
+security.enabled=true
+security.oauth2.resource.userInfoUri=@metadata-userInfoUri@
```

Comment:
need to add this in JenkinsFile

Reply:
add token info uri

Reply:
Since this property has been removed, marking the conversation as resolved